app/assets/javascripts/surveys.js

```diff
@@ -390,12 +418,23 @@
     },
 
     otherNotificationVisible: function () {
+      function isVisible (el) {
+        var style = window.getComputedStyle(el)
+        return !((style.display === 'none') || (style.visibility === 'hidden'))
```

Comment:
I've got a memory that we've checked this in previous changes by checking offsetParent which seemed a little simpler: https://github.com/alphagov/static/pull/2687/files#diff-09acb7e6d5e7bc40081c1daca83ab10c9e9ae5644fcfd06ed2438db35c002f45R49 ?